components/vf-button/vf-button.scss

```diff
@@ -4,27 +4,51 @@
   @include set-type(button--r);
 
   appearance: none;
-  border: 0;
+  border-radius: 4px;
```

Comment:
Not sure if we still need a `border: 0` before to reset any default `<button>` faff in random browser X, but I guess _if_ that's an issue we can sort later.